modules/reindex/src/test/resources/rest-api-spec/test/reindex/10_basic.yml

```diff
@@ -120,6 +121,24 @@
   - is_false: response.task
   - is_false: response.deleted
 
+  - do:
+      reindex_task:
+        task_id: $persistent_task_id
+
+  - match: {state: "done"}
+  - match: {start_time: /.+/}
+  - match: {end_time: /.+/}
+  - gte: {duration_in_millis: 0}
+  - match: {result.created: 1}
```

Comment:
Can we add an example of how the `GET _reindex/id` response looks to the PR description such that we can receive feedback on that too?

In particular, I wonder how this will interact with GET for ongoing reindex requests, where we also have created, updated etc. fields. We should probably discuss this at our next sync (but would like to base the discussion on the example).

I see no problem in finishing and merging this PR before that is clarified though.